### DISCUSSION TITLE
Does Rate Limiting applies to admins?

### DISCUSSION THREAD

**Discussion by emrahnazif** [directus/directus#16990] [Archive]
I am thinking to use rate limiter. But I would prefer admins don't hit this limits. 
[https://docs.directus.io/self-hosted/config-options.html#rate-limiting](url)

In documentation, I am not able to find this information. Does Rate Limiting applies to all users, including admins?
If we are running directus in cluster mode (with pm2), can we still use basic memory tracker or should we setup an external memcache or redis instance?

Additionally, it would be great to have role based rate limiting possibilities.

**Comment by br41nslug** [comment]
Rate limiting applies to everyone authenticated or unauthenticated because this done at the (express) server level.
When running Directus horizontally scaled you should be using a shared Redis instance(or cluster) or the cache will be out of sync between the instances (memcache wil be phased out in the near future).
